src/analysis/normed_space/lp_space.lean

```diff
@@ -88,6 +96,21 @@ lemma mem_ℓp_gen (hp : 0 < p.to_real) {f : Π i, E i} (hf : summable (λ i, 
   mem_ℓp f p :=
 (mem_ℓp_gen_iff hp).2 hf
 
+lemma mem_ℓp_gen' (hp : 0 < p.to_real) {C : ℝ} {f : Π i, E i}
```

Comment:
In the previous lemma and this one, I think you don't need the assumption `hp`.

Reply:
Ah, you're right (of course `hf` is simply impossible in these cases unless `α` is finite, so I didn't think of it.)